index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+    <title>Lessons Learned</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+    <header id="cabecalho">
+        <h1 id="titulo">Lições Aprendidas - Trybe</h1>
+    </header>
+
+    <nav id="navegacao">
+        <ul>
+            <li>Home</li>
+            <li>Login</li>
```

Comment:
@Matheusnugas normalmente os itens dentro de uma seção de navegação tem alguma forma de interação, sendo um botão, ou mais comumente, um link. Por exemplo:
```suggestion
            <li><a href="#">Home</a></li>
            <li><a href="#">Login</a></li>
```